model/connection_string.go

```diff
@@ -162,7 +163,13 @@ func (cs ConnectionString) String() string {
 		case []byte:
 			res = append(res, tag+"="+base64.StdEncoding.EncodeToString(typ))
 		case crypto.String:
-			res = append(res, tag+"="+base64.StdEncoding.EncodeToString([]byte(typ)))
+			var value string
+			if omit {
+				value = base64.StdEncoding.EncodeToString([]byte(typ[:3])) + omitted
+			} else {
+				value = base64.StdEncoding.EncodeToString([]byte(typ[:3]))
```

Comment:
```suggestion
				value = base64.StdEncoding.EncodeToString([]byte(typ))
```

Reply:
That is, if `omit` is false; shouldn't you return the entire string?

Reply:
ops, sorry